libqtile/widget/notify.py

```diff
@@ -130,7 +130,10 @@ def display(self):
         self.bar.draw()
 
     def clear(self, reason=ClosedReason.dismissed):
-        notifier._service.NotificationClosed(self.current_id, reason)
+        notifier._service.NotificationClosed(
+            notifier.notifications[self.current_id].id,
```

Comment:
Changed this as we were sending the widget's internal reference rather than the notification's id.

Reply:
Cool, thanks